Countries Info/Supporting files/AppDelegate.swift

```diff
@@ -6,31 +6,22 @@
 //
 
 import UIKit
+import Swinject
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
-
+    var window: UIWindow?
+    private let DIAssabler = DependecyInjectionManager.shared.assembler
```

Comment:
By naming convention we need to start var name from lower character, plus there is `m` letter missing, we can shorten this war to be for example just `assembler`